sqlglot/dialects/snowflake.py

```diff
@@ -76,9 +76,24 @@ def _parse_date_part(self):
     self._match(TokenType.COMMA)
     expression = self._parse_bitwise()
 
-    if this.name.upper().startswith("EPOCH"):
+    name = this.name.upper()
+    if name.startswith("EPOCH"):
+        scale = None
+
+        if name.startswith("EPOCH_MILLISECOND"):
```

Comment:
you don't need starts with here

Reply:
I did this to allow both epoch_millisecond and epoch_milliseconds